README.md

```diff
@@ -4,24 +4,62 @@
 Experimental webview editor for [nih-plug](https://github.com/robbert-vdh/nih-plug) using [wry](https://github.com/tauri-apps/wry).
 Built on top of [baseview](https://github.com/RustAudio/baseview).
 
-## Current status
+## Compatability
 
-I've only been able to test this on macOS so far on which it has been working quite robustly.
-Now that wry officially supports attaching to an existing window handle, I'm hoping that Windows also works but this needs to be verified.
+Currently, Windows and macOS are supported and have been tested.
```

Comment:
i think this claim is a bit too bold. maybe something along the lines of "and have been tentatively tested"?